### PR TITLE
Add Demo Application for Python

### PR DESCRIPTION
**Pull request by mbtamuli**
Minimal Application in Python along with accompanying Dockerfile

Signed-off-by: Mriyam Tamuli mbtamuli@gmail.com

### CODE REVIEW

File: minimal-py-app/app.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from flask import Flask, make_response
+
+app = Flask(__name__)
+
+@app.route("/errorz")
+def errorz():
+	return "Something broke!", 500
```
Comment (deepskandpal) [inline minimal-py-app/app.py]: Please add a docstring explaining what this function is accomplishing, along with the return type

File: minimal-py-app/app.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+@app.route("/healthz")
+def healthz():
+    r = make_response('{"status":"ok"}')
```
Comment (deepskandpal) [inline minimal-py-app/app.py]: lease add a docstring explaining what this function is accomplishing, along with the return type

**Review by deepskandpal — CHANGES_REQUESTED** [review]
LGTM  for the most part, Please address the comments mentioned. Additionally, it would be great if the script is run through a PEP8 standard checker to eliminate any Style guide discrepancy

File: minimal-py-app/app.py (hunk 1 above)
Comment (mbtamuli) [reply minimal-py-app/app.py]: Okay.

File: minimal-py-app/app.py (hunk 2 above)
Comment (mbtamuli) [reply minimal-py-app/app.py]: Okay.

**Comment by mbtamuli** [comment]
> Additionally, it would be great if the script is run through a PEP8 standard checker to eliminate any Style guide discrepancy

The CI is not part of this demo.

**Review by mugdha-adhav — APPROVED** [review]

**Pull request merged**